qemu/tests/cfg/ansible_test.cfg

```diff
@@ -9,3 +9,12 @@
     ansible_ssh_extra_args = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
     # Here we can define an extra set of variables for the playbook with json format
     #ansible_extra_vars = '{"debug_msg": "Hello Ansible!", "force_handlers": true}'
+    variants guest_operation:
+        - @default:
+        - responsive_migration:
```

Comment:
Need add toplevel_playbook = "test.yml" to start interactive_response on L1 guest, for me.

Reply:
Internal only.